### ISSUE TITLE
ENH: learning rate finder

### ISSUE THREAD

**Issue by Xylambda** [Xylambda/torchfitter#13]
.. [1] Sylvain Gugger - How do you find a good learning rate? 
   https://sgugger.github.io/how-do-you-find-a-good-learning-rate.html

.. [2] Leslie N. Smith - Cyclical Learning Rates for Training Neural 
   Networks
   https://arxiv.org/pdf/1506.01186.pdf

.. [3] aychang - Automated Learning Rate Suggester
   https://forums.fast.ai/t/automated-learning-rate-suggester/44199

https://gist.github.com/muellerzr/0634c486bd4939049314632ccc3f7a03
